drivers/hud_spm_report/app/models/hud_spm_report/fy2023/episode_batch.rb

```diff
@@ -49,16 +80,17 @@ def calculate_batch(client_ids)
     end
 
     # The associations seem to make imports run one at a time, so, they are passed separately in parallel arrays
-    private def save_episodes!(episodes, bed_nights, enrollment_links)
+    private def save_episodes!(episodes, _bed_nights, enrollment_links)
```

Comment:
We're leaving the bed_nights var because we might want to enable this in the future?

Reply:
Yes, we disabled it to prevent saving hundreds of thousands of rows that aren't currently displayed in the UI, but wanted to leave the pattern behind in case we want to re-enable that in the future.